styleguide/utils.py

```diff
@@ -27,3 +27,19 @@ def get_styleguide_dirs():
                 if os.path.isdir(str(template_dir)):
                     template_dirs.append(str(template_dir))
     return template_dirs
+
+def get_example_sections(example):
+    """Parses a multipart example and returns them in a dictionary by type.
+
+    Types will be by language to highlight, except the special "__doc__"
+    section. The default section is "html".
+    """
+
```

Comment:
PR looks great on the layout page.  Other pages that use get_example_sections() (i.e. form, typography) are currently giving the key error.  Obviously there are no commented out section titles in those examples.  Should this function return something useful (like the example html that is passed in) in that case and default to html as the type?  Or are we relying on us to always include those titles in our html?

Reply:
that was the intent. Fixing it now.